### DISCUSSION TITLE
Getting "Support for password authentication was removed on August 13, 2021." even though I am typing in a Personal access token?

### DISCUSSION THREAD

**Discussion by shaunakshende** [community/community#114406] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Is anyone else experiencing the following issue? Up until 2 days ago, when I pushed I would enter my username and copy/paste my personal access token. Now, when I paste my PAT, I get the following message:

"Support for password authentication was removed on August 13, 2021." 

I tried generating a new PAT and entering it, still getting the same message. It's not recognizing the PAT. I have also checked that I am not appending characters when copy/pasting....

**Comment by devashish2024** [comment]
It is, yes, removed. Nothing else to talk about, Idk why too

**Reply by shaunakshende** [reply]
I am attempting to authenticate using a personal access token, not a password. I am still getting this message.

**Reply by chevalierid** [reply]
I'm getting this problem as well. The personal access token was working fine then stopped a couple of weeks ago. I have not tried generating a new PAT; unfortunate to hear that didn't help you. Very frustrating issue!

**Reply by chevalierid** [reply]
Not sure if this is helpful but I followed these steps and pushing appears to work again (for now):

1. configured the global email and username
2. generated a new token
3. reset the remote URL for origin using `git remote set-url origin https://<token>@github.com/<user>/<reponame>.git`
4. stored the PAT using `git config --global credential.helper store`

**Reply by amvasko** [reply]
> Not sure if this is helpful but I followed these steps and pushing appears to work again (for now):
> 
>     1. configured the global email and username
> 
>     2. generated a new token
> 
>     3. reset the remote URL for origin using `git remote set-url origin https://<token>@github.com/<user>/<reponame>.git`
> 
>     4. stored the PAT using `git config --global credential.helper store`

Awesome, thanks, great job. It works for me ^_^
